### PR TITLE
evscript: git-47f86f0 -> unstable-2021-06-16

### PR DESCRIPTION
**Pull request by MilesBreslin**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/inputmethods/evscript/default.nix [hunk 1]
```diff
@@ -7,11 +7,11 @@ rustPlatform.buildRustPackage rec {
...
-    sha256 = "1xm8297k0d8d0aq7fxgmibr4qva4d02cb6gnnlzq77jcmnknxv14";
+    rev = "410603abf4810320bf79bde90cc85367b71a8b12";
+    sha256 = "sha256-apq3kHipEX6zOTNwqpIQR46JqmeE7EKVSOGrNNSkyu8=";
   };
 
-  cargoSha256 = "1z0xxbjnhhzn1fnc3zhvy727l88qyyfqdayj5yvf3nh23m7sr87l";
+  cargoHash = "sha256-5PWyxUfE6lkQ8CYXxq2/xp8hr5k73ZpK2RVJCgaWX1c=";
 
   meta = with lib; {
     homepage = "https://github.com/myfreeweb/${pname}";
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/evscript/default.nix:17]:
```suggestion
    homepage = "https://github.com/myfreeweb/evscript";
```

File: pkgs/tools/inputmethods/evscript/default.nix [hunk 2]
```diff
@@ -7,11 +7,11 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/evscript/default.nix:7]: Please set the version to unstable-XXXX-XX-XX where XXXX-XX-XX is the date of the commit.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by MilesBreslin** [comment]
@SuperSandro2000

File: pkgs/tools/inputmethods/evscript/default.nix [hunk 3]
```diff
@@ -2,19 +2,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "evscript";
-  version = "git-${builtins.substring 0 7 src.rev}";
+  version = "git-2021-06-16";
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/evscript/default.nix:5]:
```suggestion
  version = "unstable-2021-06-16";
```

File: pkgs/tools/inputmethods/evscript/default.nix [hunk 4]
```diff
@@ -2,19 +2,19 @@
...
   pname = "evscript";
-  version = "git-${builtins.substring 0 7 src.rev}";
+  version = "unstable-2021-06-16";
 
   src = fetchFromGitHub {
     owner = "myfreeweb";
     repo = pname;
-    rev = "47f86f0d15add2af785ea1ff47f24d130026d1b4";
-    sha256 = "1xm8297k0d8d0aq7fxgmibr4qva4d02cb6gnnlzq77jcmnknxv14";
+    rev = "410603abf4810320bf79bde90cc85367b71a8b12";
```
Comment (SuperSandro2000) [inline pkgs/tools/inputmethods/evscript/default.nix:10]: It is not allowed to directly fetch commits from a fork. Please fetch from the main repo and add your changes as patch(es).

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by MilesBreslin** [comment]
@SuperSandro2000

**Comment by MilesBreslin** [comment]
@SuperSandro2000

**Pull request merged**